internal/persistence/persistors/mysqlstore/store_organization.go

```diff
@@ -0,0 +1,241 @@
+	if params == nil {
+		return nil, ErrOrgNil
+	}
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return nil, fmt.Errorf("extract context executor: %v", err)
+	}
+
+	entry := &mysqlmodel.Organization{ID: params.Id}
+	cols := []string{mysqlmodel.OrganizationColumns.ID}
+
+	//if params.OrganizationTypeRefId.Valid {
+	//    entry.OrganizationRefUsers() = params.OrganizationTypeRefId.Int
+	//    cols = append(cols, mysqlmodel.OrganizationColumns.IsActive)
+	//}
+
+	if params.Name.Valid {
+		entry.Name = params.Name.String
+		cols = append(cols, mysqlmodel.OrganizationColumns.Name)
+	}
+
+	_, err = entry.Update(ctx, ctxExec, boil.Whitelist(cols...))
+	if err != nil {
+		return nil, fmt.Errorf("update failed: %v", err)
+	}
+
+	organization, err := m.GetOrganizationById(ctx, tx, entry.ID)
+	if err != nil {
+		return nil, fmt.Errorf("get organitzation by id: %v", err)
+	}
+
+	return organization, nil
+}
+
+func (m *Repository) GetOrganizationById(ctx context.Context, tx persistence.TransactionHandler, id int) (*model.Organization, error) {
+	paginated, err := m.GetOrganizations(ctx, tx, &model.OrganizationFilters{
+		IdsIn: []int{id},
+	})
+	if err != nil {
+		return nil, fmt.Errorf("organization filtered by id: %v", err)
+	}
+
+	if paginated.Pagination.RowCount != 1 {
+
+		return nil, fmt.Errorf(sysconsts.ErrExpectedExactlyOneEntry, id)
+	}
+
+	return &paginated.Organizations[0], nil
+}
+
+func (m *Repository) GetOrganizations(ctx context.Context, tx persistence.TransactionHandler, filters *model.OrganizationFilters) (*model.PaginatedOrganization, error) {
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return nil, fmt.Errorf("extract context executor: %v", err)
+	}
+
+	res, err := m.getOrganizations(ctx, ctxExec, filters)
+
+	if err != nil {
+		return nil, fmt.Errorf("read organizations: %v", err)
+	}
+
+	return res, nil
+}
+
+func (m *Repository) getOrganizations(ctx context.Context,
+	ctxExec boil.ContextExecutor,
+	filters *model.OrganizationFilters) (*model.PaginatedOrganization, error) {
+
+	var (
+		paginated  model.PaginatedOrganization
+		pagination = model.NewPagination()
+		res        = make([]model.Organization, 0)
+		err        error
+	)
+
+	ctx, cancel := context.WithTimeout(ctx, m.cfg.QueryTimeouts.Query)
+	defer cancel()
+
+	queryMods := []qm.QueryMod{
+		qm.Select(
+			fmt.Sprintf("%s.%s AS %s",
+				mysqlmodel.TableNames.Organization,
+				mysqlmodel.OrganizationColumns.ID,
+				mysqlmodel.OrganizationColumns.ID,
+			),
+			fmt.Sprintf("%s.%s AS %s",
+				mysqlmodel.TableNames.Organization,
+				mysqlmodel.OrganizationColumns.Name,
+				mysqlmodel.OrganizationColumns.Name,
+			),
+			fmt.Sprintf("%s.%s AS %s",
+				mysqlmodel.TableNames.Organization,
+				mysqlmodel.OrganizationColumns.IsActive,
+				mysqlmodel.OrganizationColumns.IsActive,
+			),
+		),
+	}
```

Comment:
As mentioned [above](https://github.com/dembygenesis/theoverwatchtools.cli/pull/31/files#r1702885319), you might want to update this query to include more columns

Reply:
The query has now been altered to include more columns.